### PR TITLE
Discard support of some deprecated ns extensions

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
Only  &v3_ns_ia5_list[ 0...6 ] are used.

### CODE REVIEW

**Comment by richsalz** [comment]
I think it's better to just purge this v3_ns stuff completely.

File: apps/openssl-vms.cnf [hunk 1]
```diff
@@ -188,7 +188,7 @@ basicConstraints=CA:FALSE
 # keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 
 # This will be displayed in Netscape's comment listbox.
-nsComment			= "OpenSSL Generated Certificate"
+#nsComment			= "OpenSSL Generated Certificate"
```
Comment (kroeckx) [inline apps/openssl-vms.cnf]: Maybe you should just remove the whole comment thing?

**Comment by FdaSilvaYY** [comment]
Should I purge documentation about `DEPRECATED EXTENSIONS` and co :
- from  doc\apps\x509v3_config.pod 
- from object.txt ?

File: apps/openssl-vms.cnf (hunk 1 above)
Comment (snhenson) [inline apps/openssl-vms.cnf]: It also means that this would be an OpenSSL 1.2 change.

**Comment by richsalz** [comment]
keep it in objects.txt (lots of crap there), but do 'git grep nsComment' and you will see that there is no code to parse this at all any more.

**Comment by richsalz** [comment]
so @FdaSilvaYY what's up with this?  remove the WIP+rebase or close this, okay?

**Comment by FdaSilvaYY** [comment]
@richsalz  Rebased with some rewording. Should we update the CHANGES file?

**Comment by richsalz** [comment]
yes, add a comment to CHANGES.  ping @kroeckx ; if he reviews i'll merge.

**Comment by FdaSilvaYY** [comment]
Done, and rebased !

**Comment by levitte** [comment]
@richsalz, you labeled 1.1.0, is there any reason why this shouldn't go to master as well?

**Comment by FdaSilvaYY** [comment]
Rebased, and a bit re-worded ;) !

**Comment by FdaSilvaYY** [comment]
IMHO, I think that pushing a warning about these deprecated extensions would be valuable to the user...
to help him discarding these extensions from config files.
Somewhere/somehow

**Comment by FdaSilvaYY** [comment]
I add a new commit to push an error, when parsing or serializing this type of extension.
Ping @openssl :  Any comments or advice on how to conclude this PR ?

**Comment by mattcaswell** [comment]
Ping @openssl. What do we do with this?

**Comment by richsalz** [comment]
Move it to 1.2.0

**Comment by FdaSilvaYY** [comment]
Hin
The initial version of this PR was removing just one unused extra-cell of `v3_ns_ia5_list` array.
Do you want a PR for it only, so it can be part of 1.1.1 ?

File: CHANGES [hunk 2]
```diff
@@ -427,6 +427,11 @@
      issues.
      [Matt Caswell]
 
+  *) Discard support of some obsolete X509.v3 Netscape (NS) extensions.
+     Except nsComment who is still present in openssl.cfg files and co.
+     See the list of these old extensions in doc/man5/x509v3_config.pod 
+     [FDaSilvaYY]
+
```
Comment (levitte) [inline CHANGES]: This needs to be moved up

**Comment by richsalz** [comment]
You're not worried about breaking compatibility?  Think the ncComment is enough?

**Comment by levitte** [comment]
We could conduct a poll on openssl-users...

File: CHANGES (hunk 2 above)
Comment (FdaSilvaYY) [reply CHANGES]: Lazy conflict fixup ;)

**Comment by FdaSilvaYY** [comment]
Closed in favor of #9241 9241

**Pull request closed without merging**